### PR TITLE
Remove codecov references

### PR DESCRIPTION
**Pull request by kacperkaminski-st**
## Overview

Remove codecov references as we do not use it anymore. Part of [codecov removal ticket](https://github.com/sensortower/overseer/issues/20854)

## Acceptance criteria

- [ ] CI works fine

## Key stakeholders
- @IvanTakarlikov-st 
- @sensorsasha 
- @muhammad-st

### CODE REVIEW

**Review by muhammad-st — APPROVED** [review]
LGTM

Let's also wait for an approval from @sensorsasha

**Review by sensorsasha — APPROVED** [review]

**Pull request merged**
